packages/libretro/retroarch/package.mk

```diff
@@ -19,7 +19,7 @@
 ################################################################################
 
 PKG_NAME="retroarch"
-PKG_VERSION="d360265"
+PKG_VERSION="91744d9"
```

Comment:
May or may not need to force a `--disable-qt`? Probably just sees Qt is not there, and then disables it automatically.

Reply:
https://github.com/libretro/RetroArch/commit/19f6798947f8a4dcbfcc43aab6d30686dfbf8e54